app/models/health/qualifying_activity.rb

```diff
@@ -284,6 +292,8 @@ def mode_of_contact_other_value
     end
 
     def reached_client_is_collateral_contact?
+      return false if from_epic?
```

Comment:
reached_client_is_collateral_contact is referenced in views:
 app/views/health/sdh_case_management_notes/_activities.haml
app/models/health/qualifying_activity.rb